Extension/Twig/PaginationMetaExtension.php

```diff
@@ -39,11 +35,17 @@ class PaginationMetaExtension extends \Twig_Extension
     protected $accessor;
 
     /**
+     * @var ContainerInterface
+     */
+    protected $container;
+
+    /**
      * PaginationMetaExtension constructor.
      * @param RouterInterface $router
      */
-    public function __construct(RouterInterface $router)
+    public function __construct(RouterInterface $router, ContainerInterface $container)
```

Comment:
Isn't necessary to inject symfony container, you can inject only `%skuola_seo.bad_route_info_params%`.